.gitlab/corona-templates.yml

```diff
@@ -19,7 +19,8 @@
       when: never
     - if: '$CI_JOB_NAME =~ /release_resources/'
       when: always
-    - when: on_success
```

Comment:
This rule applies to all corona jobs, so I suggest the following strategy for rules:
- first: define all rules that prevent those jobs from running
- then: define all rules specific to one job (here release_resources)
- finally: conclude with a default rules that allows the job to run `on_success` in all other circumstances.

This is because gitlab won’t run the job if there is not true rule and because it is easier to stack "when: never" rules.